test/index.js

```diff
@@ -20,6 +20,13 @@ import { getAllFiles, getAllFilesSync } from '../src/index.js'
 
 const fixtures = `./test/fixtures`
 
+const isInList = list => name => list.indexOf(name) > -1
```

Comment:
I think this can be more simply written as `list => name => list.includes(name)`